File: SuperP/app/src/main/java/com/zsy/superp/java/util/LogUtils.java
package com.zsy.superp.java.util;

import android.content.Context;
import android.util.Log;
import android.widget.TextView;

import org.w3c.dom.Text;

import java.util.ArrayList;
import java.util.List;

public class LogUtils {

    private static String Tag = "Java-LogUtils";
    private static boolean mIsPrint = true;

    public static void setIsPrint(boolean isPrint){
        mIsPrint = isPrint;
    }

    public static void e(String msg){
        if(mIsPrint) Log.e(Tag,msg);
    }
    public static void e(String tag,String msg){
        if(mIsPrint) Log.e(tag,msg);
    }

}

File: SuperP/app/src/main/java/com/zsy/superp/kt/util/LogUtils.kt
package com.zsy.superp.kt.util

import android.util.Log

object LogUtils {

    var switch = false
    var target = "LogUtils"

    fun switch(isClose : Boolean){
        switch = isClose
    }

    fun e(msg:String){
        if(!switch) Log.e(target,msg)
    }
    fun e(tar:String,msg:String){
        if(!switch) Log.e(tar,msg)
    }
}